### ISSUE TITLE
post-processing animation

### ISSUE THREAD

**Issue by xwa9860** [pyrk/pyrk#13]
make a nice animation for the pebble temperature evolution during transients.
(expand to a more general tool in the future)

**Comment by katyhuff** [comment]
cool. good idea. I'd love to see this be an interactive type of animation (which can be done a little easier if we store the output data in a data base rather than just in memory/logfiles and use an interactive plotting tool such as [bokeh](http://bokeh.pydata.org/en/latest/).

**Comment by xwa9860** [comment]
Should we store the data with some numpy function like numpy.save?

**Comment by katyhuff** [comment]
That's certainly one straightforward and quick option. That said, I would strongly prefer a more robust solution, using a backend database in either hdf5 (with pytables or h5py) or sqlite (with sqlite3).  I started to implement that at some point (I deleted those db package files in the reorg https://github.com/pyrk/pyrk/pull/6/files) and put it on the backburner. Let's move the database discussion to another issue, as it's a separate issue in many respects (I have now created issue #16). It would be great to talk through the data model together, if you have some time this week or next week.  That said, ideally this issue should only be pursued once pull request #10 is complete, as it is a higher priority.
